# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/curl/hotfix-glibc-236.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- curl-7.84.0/lib/easy_lock.h.vanilla	2022-08-04 23:27:57.674330576 +0200
+++ curl-7.84.0/lib/easy_lock.h	2022-08-04 23:28:19.218329717 +0200
@@ -24,6 +24,8 @@
 
 #include "curl_setup.h"
 
+#include <pthread.h>
+
 #define GLOBAL_INIT_IS_THREADSAFE
 
 #if defined(_WIN32_WINNT) && _WIN32_WINNT >= 0x600
